packages/jss/src/plugins/rules.js

```diff
@@ -30,4 +31,27 @@ const plugins: $ReadOnlyArray<Plugin> = Object.keys(classes).map((key: string) =
   return {onCreateRule}
 })
 
+// Animation name ref replacer.
+plugins.push({
```

Comment:
Not a fan of this. I think we should rename all of the Rules to StylePlugin etc. and move this functionality to the KeyframesPlugin.

Reply:
yeah, that would be cleaner, but also more hooks and more code, not sure its worth it

Reply:
I think it's worth it.